pkgs/tools/networking/xxh/default.nix

```diff
@@ -0,0 +1,19 @@
+{ lib, fetchFromGitHub, buildPythonApplication, pexpect, pyyaml, openssh }:
+buildPythonApplication rec{
+  pname = "xxh";
+  version = "0.8.7";
+  src = fetchFromGitHub {
+    owner = pname;
+    repo = pname;
+    rev = version;
+    hash = "sha256-AKfiFBaV8DC/Z7Bc+ZpwcJor/mzYomUaQKKobKXICn4=";
+  };
+  propagatedBuildInputs = [ pexpect pyyaml openssh ];
```

Comment:
Can we do either or both checkInputs and pythonImportsCheck?

Reply:
I dunno, its test suite requires docker compose (https://github.com/xxh/xxh/blob/master/xde/docker-compose.yml).

Reply:
We should be able to do 

```
pythonImportsCheck = [ "xxh" ];
```
or 
```
pythonImportsCheck = [ "xxh.xxh" ];
```

Reply:
Build fails with no module called xxh or xxh.xxh

Reply:
Similar issue arise with xde.
At the end of the day the project itself is already running CI tests and it's python, so I'm not too worried

Reply:
I've written a patch `http://0x0.st/osV-.patch` to write a nixos test. It can be ran with `nix-build -A xxh.tests`